@xen-orchestra/lite/src/App/TreeView.tsx

```diff
@@ -41,6 +46,17 @@ const getIconColor = (obj: Host | Vm) => {
 
 const TreeView = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
   {
+    initialState: ({ location }) => ({
+      _selectedNodes: location.pathname.split('/')[3] !== undefined ? [location.pathname.split('/')[3]] : [],
+    }),
+    effects: {
+      setSelectedNodes: function (_, nodeIds) {
+        if (this.state.objectId === undefined) {
+          window.location.hash = '/infrastructure/type/' + nodeIds[0]
+        }
```

Comment:
Why do you need this?

Reply:
It's needed to trigger the "select processus" for a node.